sound/soc/sof/sof-priv.h

```diff
@@ -207,6 +207,7 @@ struct snd_sof_dsp_ops {
 	/* pre/post firmware run */
 	int (*pre_fw_run)(struct snd_sof_dev *sof_dev); /* optional */
 	int (*post_fw_run)(struct snd_sof_dev *sof_dev); /* optional */
+	int (*parse_hda_ext_manifest)(struct snd_sof_dev *sof_dev); /* optional */
```

Comment:
is this really HDA specific? This is a generic SOF DSP structure, cannot non-HDA implementations contain custom manifest data too?

Reply:
maybe call this "parse_platform_ext_manifest"? this seems to be the chosen wording in https://github.com/thesofproject/sof/pull/2893

So in sof-priv.h and loader, only refer to "platform_ext_manifest" and then in sof/intel/, the implementation can be named a hda specific one.

Reply:
all ops are already platform-specific. So something like parse_ext_manifest_config_data() is more suitable

Reply:
also, typically for all ops, we provide a wrapper in ops.h. Can you add the op and its wrapper in ops.h in a separate patch before defining it for HDA platforms?